.browserslistrc

```diff
@@ -0,0 +1 @@
+defaults
```

Comment:
add this file to .gitignore and run command: git rm --cached .browserlistrc to untrack it
follow steps below:
Step 1: Open file: .gitignore and add this line: .browserlistrc (same as line 44 in bellow picture)
Step 2: Open terminal and cd to your project. Then run command: git rm --cached .browserlistrc
That's all !

Reply:
done